Automattic/wp-calypso
# AT: cleanup redux, show wp-admin for AT

- Removes "PRESSABLE" space from redux which was introduced long time ago to track AT sites, but turned out not necessary
- Handle new property of /me/sites that indicates pressable sites
- Show WPadmin link for AT sites
- Resolves 50-gh-automated-transfer-api-contracts

## Testing instructions
- Sandbox D4210-code
- Pick AT test site
- See that wp-admin is there
- Check on prod some other site that does not show wpadmin (user registered after '2015-09-07' or jetpack)
- See that branch does not show jetpack

CC @lamosty @gwwar @rralian
